src/components/themed/WalletList.tsx

```diff
@@ -314,6 +314,12 @@ export const filterWalletCreateItemListBySearchText = (createWalletList: WalletC
       out.push(item)
     }
   }
+  out.push({
```

Comment:
It is not necessary to add a new row. Instead, you can apply `contentContainerStyle={{ paddingBottom: theme.rem(4.25) }}` to create one row's worth of extra space at the bottom of the scroll area. So basically, this entire commit becomes a single-line style addition.

Reply:
will do